core/client/fs/src/main/java/alluxio/client/block/stream/DefaultBlockWorkerClient.java

```diff
@@ -167,6 +169,12 @@ public RemoveBlockResponse removeBlock(final RemoveBlockRequest request) {
         .removeBlock(request);
   }
 
+  @Override
+  public MoveBlockResponse moveBlock(MoveBlockRequest request) {
+    return mRpcBlockingStub.withDeadlineAfter(mDataTimeoutMs, TimeUnit.MILLISECONDS)
+        .moveBlock(request);
```

Comment:
I don't think this is the right timeout to use. The MoveBlock request could take a significantly longer period of time due to data movement.

Reply:
do you have a recommended timeout value in this case?

Reply:
Not any existing value, take a look at the worker side move block call and see what kind of upper bound it has in terms of the retries.

Reply:
the default value for `mDataTimeoutMs` is 30 secs and the default value for worker side move block call timeout is 10 secs.   Maybe we can use this value if the default is that high?